cli/src/main/java/org/commonjava/maven/ext/cli/Cli.java

```diff
@@ -448,10 +449,10 @@ private void createSession( File target, File settings )
             {
                 // No, this is not a typo. If current default is null, supply new local and global.
                 // This function passes in settings to make it easier to test.
-                this.settings = settings = new File( System.getProperty( "user.home" ), ".m2/settings.xml" );
+                this.settings = settings = new File( System.getProperty( "user.home" ), ".m2" + File.separator + "settings.xml" );
```

Comment:
This is obviously a fix for file separator not URL.

Reply:
I've changed this code so I think it might be good to rebase to master please